packages/ui/src/components/toggle-group.tsx

```diff
@@ -142,21 +147,32 @@ const ToggleGroupItem = forwardRef<
   ElementRef<typeof ToggleGroupPrimitive.Item>,
   ComponentPropsWithoutRef<typeof ToggleGroupPrimitive.Item> & ToggleGroupItemProps
 >(({ value, tooltipProps, disabled: itemDisabled, iconOnly, children, ...props }, ref) => {
-  const { variant, size, disabled: groupDisabled = false, selectedValues } = useContext(ToggleGroupContext)
+  const {
+    variant,
+    selectedVariant,
+    size,
+    disabled: groupDisabled = false,
+    selectedValues
+  } = useContext(ToggleGroupContext)
 
   const buttonVariant = useMemo(() => {
-    const [inactiveVariant, activeVariant] = variant.split('-') as [ButtonVariants, ButtonVariants]
     const isPressed = !!selectedValues.get(value)
```

Comment:
NIT: can we change the variable name as `isSelected`? So that it will be in sync with the API.